packages/koenig-lexical/test/utils/e2e.js

```diff
@@ -108,6 +111,10 @@ export function prettifyHTML(string, options = {}) {
         output = document.body.innerHTML;
     }
 
+    if (options.ignoreDataCardWidth) {
+        output = output.replace(/data-card-width="[^"]*"/g, '');
+    }
```

Comment:
I think we need to be selective with the options that are added to this helper otherwise it's going to become unwieldy. In this case I'm not seeing a compelling case for ignoring a single specific data attribute, we should stick to options that ignore large blocks of generic content that aren't useful in specific test cases.